### ISSUE TITLE
Insufficient revision information for QA

### ISSUE THREAD

**Issue by sorooris** [trezor/trezord-go#203]
**Describe the bug**
Running the standard local address + port displays only the version, not revision, which is relevant and important for dev + QA cooperation.

**App version and revision**
Bridge 2.0.30 RC5 (apparent here)

**Setup**:
OS: MacOS Catalina 10.15.7

**Steps to reproduce:**

1. Open the standard Bridge local address + port 127.0.0.1:21325
2. Observe.

**Expected behavior:**
Clickable link to commit hash (similar to Suite) is present.

**Actual behavior:**
Only general version (2.0.30) is displayed on the page.

Screenshot:
![image](https://user-images.githubusercontent.com/10185314/98576153-6e04c580-22ba-11eb-8f8f-64e940e9ddc2.png)

**Comment by prusnak** [comment]
Partially addressed in 9dd2522d4492f1d89d252b8782e9ca8932de38d6

We still need to update release scripts to override githash during build via

```
go build -ldflags="-X 'main.githash=$(git rev-parse --short HEAD)'"
```

**Comment by prusnak** [comment]
More stuff added in 1c5405b6aea08e6a1dd39704a0f923d2e6d992f1 which closes the issue
